packages/backend/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "./lib",
-    "target": "ES2020",
+    "target": "ES2022",
```

Comment:
Is that somehow connected to this feature?

Reply:
No